src/ducks/Studio/FeesDistribution/FeesDistribution.js

```diff
@@ -51,22 +52,21 @@ const useFeeDistributions = ({ from, to }) => {
   }
 }
 
-export const FeesDistributionTitle = ({ setInterval }) => {
-  return (
-    <BlockHeader
-      setInterval={setInterval}
-      defaultIndex={1}
-      ranges={FEE_RANGES}
-      title='Fees Distribution'
-    />
-  )
-}
-
-const FeesDistribution = ({ onDisable }) => {
+const FeesDistribution = ({
+  onDisable,
+  deleteConnectedWidget,
+  widget,
+  parentWidget,
+  ...rest
+}) => {
   const [interval, setInterval] = useState('1d')
-  const [customDate, setCustomDate] = useState(null)
+  const setCustomDate = useState(null)[1]
```

Comment:
Looks like useless set state here, because state of this not using anywhere. And if [`onChangePeriod`](https://github.com/santiment/app/pull/2084/files#diff-496af6a0c15eb8a46989dec642ad34e61fbda9c5a39dce4cdb8e4a248c8ce212R95) not used, you can delete it